cypress/e2e/ui/Alert-add-success.cy.js

```diff
@@ -0,0 +1,42 @@
+import '@testing-library/cypress/add-commands';
+
+import { getUKFormattedDate } from '../../../src/utils/helpers';
+
+describe('ALERT ADD EXPENSE - SUCCESS', () => {
+  before(() => {
+    cy.intercept('GET', '**/items*', []).as('getExpenses');
+    cy.visit('/');
+    cy.wait('@getExpenses');
+  });
+
+  beforeEach(() => {
+    cy.intercept('POST', '**/items*', { fixture: 'expense-post' }).as(
+      'postExpense'
+    );
+  });
+
+  it('adds the expense', () => {
+    cy.addExpenseFromFixture('expense-raw');
```

Comment:
i found it confusing that the actual start of a test is hidden in a before filter. i think they should only contain setup-related stuff. can we maybe have something (e.g. a command) that we do explicitly at the start of a test? e.g. 

```javascript
 it('adds the expense', () => {
    visitHomePage();
    cy.addExpenseFromFixture('expense-raw');
    cy.wait('@postExpense');
  });

Reply:
I didn't use a command just passed the `cy.visit('/')` and `cy.await()` for the different tests. It's just two lines of code and stays flexible.